packages/common/src/core/types/IHubItemEntity.ts

```diff
@@ -191,4 +191,6 @@ export type IHubItemEntityEditor<T> = Omit<T, "extent"> & {
    * catalog with a new group or an existing group.
    */
   _catalogSetup?: ICatalogSetup;
+
+  _layoutSetup?: Record<string, unknown>;
```

Comment:
can we make this more explicit: 

`_layoutSetup?: {
  layout: "blank" | "simple"
}`

we can always extend it in the future and/or pull it out into a separate type/interface.